lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -45,43 +46,32 @@ def call(params)
           results = []
 
           client = params.gdc_gd_client
-          development_client = params.development_client
-
-          synchronize = params.synchronize.map do |info|
-            from_project = info.from
-            to_projects = info.to
-
-            from = development_client.projects(from_project) || fail("Invalid 'from' project specified - '#{from_project}'")
-            params.gdc_logger.info "Creating Blueprint, project: '#{from.title}', PID: #{from.pid}"
-
-            blueprint = from.blueprint(include_ca: include_ca)
-            info[:to] = to_projects.pmap do |entry|
-              pid = entry[:pid]
-              to_project = client.projects(pid) || fail("Invalid 'to' project specified - '#{pid}'")
-
-              params.gdc_logger.info "Updating from Blueprint, project: '#{to_project.title}', PID: #{pid}"
-              ca_scripts = to_project.update_from_blueprint(
-                blueprint,
-                update_preference: params.update_preference,
-                execute_ca_scripts: false,
-                exclude_fact_rule: exclude_fact_rule
-              )
-
-              entry[:ca_scripts] = ca_scripts
-
-              results << {
-                from: from_project,
-                to: pid,
-                status: 'ok'
-              }
-              entry
-            end
-
-            info
+          # TODO: ^^^^ de-duplicated
+
+          all_results = []
+          synchronize = []
+          params.synchronize.each do |segment|
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)